Packages/com.unity.inputsystem/InputSystem/Controls/InputControl.cs

```diff
@@ -873,6 +874,18 @@ internal bool isSetupFinished
             }
         }
 
+        internal bool isButton
+        {
+            get => (m_ControlFlags & ControlFlags.IsButton) == ControlFlags.IsButton;
+            set
+            {
```

Comment:
Setter on question-as-property feels weird, `control.isButton = true`  is hard to understand what it is gonna to. Either we need a better prop, like `controlType` which accepts an enum, and then that is mapped to flags internally. Or remove setter completely, because there is only one place in this PR using the setter

Reply:
Maybe it's ok for now because button is a property of control, not type of control. But it's even more confusing.